### ISSUE TITLE
Detect Existing Sidecar

### ISSUE THREAD

**Issue by dcramer** [getsentry/spotlight#324]
Let's add a signature (e.g. HEAD http://localhost:8369, something like X-Powered-By) so that Spotlight can detect an already running sidecar rather than retrying in a loop infinitely. We might still want to retry, but we can at least not log it, and probably retry at a slower interval.

**Comment by Shubhdeep12** [comment]
Hi @dcramer,

Currently, we're terminating the process upon detecting an existing sidecar. Is this approach satisfactory? If so, we can consider the issue resolved. If not, let me know, If there are considerations or alternative perspectives regarding our current approach.

Thanks.

**Comment by dcramer** [comment]
Yep IMO thats fine - as long as we make it so its not cryptic to the end-user (in case they need to debug).
